@ndlib/gatsby-theme-marble/src/components/Common/form.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import InputText from '../Common/inputText'
+import InputTextRadio from '../Common/inputRadio'
+import Recaptcha from 'react-google-invisible-recaptcha'
+
+import style from './style.module.css'
+import InputBox from './inputBox';
+
+class Form extends Component {
+  state = { 
+    data: {},
+    errors: {},
+    value: '',
+   }
+
+   validate = () => {
+    const errors = {}
+
+    const { data } = this.state
+
+    if (data.email.trim() === '')
+      errors.email = 'Email is required'
+    if (!data.email.includes('@')) 
+      return 'Email requires an @'
+    if (data.feedback.trim() === '')
+      errors.feedback = 'Please enter your comment or question'
+    if (data.category.trim() === '')
+      errors.category = 'A selection is required'
+
+    return Object.keys(errors).length === 0 ? null : errors
+  }
+
+  validateProperty = ({ name, value }) => {
+    if (name === 'email') {
+      if (value.trim() === '') return 'Email is required'
+      if (!value.includes('@')) return 'Email requires an @'
```

Comment:
I know we don't exactly have a standardized style guide, but in practice I'm under the impression single-line ifs are discouraged (for javascript, anyway.) The return should be on it's own line, and I would go so far as to include braces despite it being a single statement, but I'm less of a stickler on that one. :)

Also, the `validate` function duplicates this logic. I would consider how you could refactor this to not repeat the same logic.